model/database/registry.yaml

```
@@ -114,7 +114,11 @@ groups:
 
           This attribute has stability level RELEASE CANDIDATE.
         examples: ["102", "ORA-17002", "08P01", "404"]
-
+      - id: db.response.row_count
+        type: int
+        stability: experimental
+        brief: Number of rows returned by the database in response to a query.
```

Not sure if query is necessary here (some databases don't have a notion of query, but they do return items)

```suggestion
        brief: Number of rows returned by the database in response.
```